src/configs/chain_configs/comdex-testnet.json

```diff
@@ -0,0 +1,155 @@
+{
+  "title": "Comdex Block Explorer",
+  "network": "meteor-test",
+  "icon": "/comdex/images/comdex/icon.svg",
+  "logo": {
+    "default": "/comdex/images/comdex/logo.svg"
+  },
+  "prefix": {
+    "consensus": "comdexvalcons",
+    "validator": "comdexvaloper",
+    "account": "comdex"
+  },
+  "genesis": {
+    "time": "2021-10-15T12:58:04",
+    "height": 4078001
+  },
+  "primaryTokenUnit": "ucmdx",
+  "votingPowerTokenUnit": "ucmdx",
+  "tokenUnits": {
+    "ucmdx": {
+      "display": "cmdx",
+      "exponent": 6
+    }
+  },
+  "extra": {
+    "profile": true
+  },
+  "endpoints": {
+    "graphql":"https://gql-testnet.comdex.forbole.com/v1/graphql",
+    "graphqlWebsocket":"wss://gql-testnet.comdex.forbole.com/v1/graphql",
+    "publicRpcWebsocket":"ws://65.108.130.132:26657/websocket"
```

Comment:
```suggestion
    "publicRpcWebsocket":"https://meteor.rpc.comdex.one/websocket"
```

Reply:
@MonikaCat  seems like the ws of this rpc endpoint `meteor.rpc.comdex.one` is not working :'(